rfps/rfp-consensus-visualization-tool.md

```diff
@@ -0,0 +1,123 @@
+# RFP: Consensus Visualization Tool
+
+**Name of Project:** `Consensus Visualization Tool`
+
+**Proposal Category:** `app-dev`
+
```

Comment:
Can you also add a Proposer here? This was in the original template, which I realized got deleted. Will re-upload.

Reply:
I added some contact information at the end of the proposal. Is that sufficient? The idea for this also came from frrrist and there were contributors to the other proposal as well.